scripts/ci.publish.js

```diff
@@ -40,11 +40,7 @@ function projectSetup() {
 function prePublishToNpm() {
   logSection('Prepublish');
 
-  log('Gathering up iOS artifacts...');
-  process.chdir('detox');
-  const {packageIosSources} = require('../detox/scripts/pack_ios');
-  packageIosSources();
-  process.chdir('..');
+  //No longer need to gather up iOS artifacts here, as those are created in a child build on CI.
 }
```

Comment:
well if there's nothing left here, can it be removed?

Reply:
I don't know who calls it and didn't want to investigate. All of this will change soon with Next anyway.

Reply:
Not sure what you mean. This is an internal func used only once here in the script...

Reply:
Cool, removed.